Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -174,6 +206,33 @@ define([
         });
     }
 
+    function requestImage(imageryProvider, col, row, level, request, interval) {
+        var dynamicIntervalData = defined(interval) ? interval.data : undefined;
+        var resource = imageryProvider._tileProvider._resource; // We actually want to set the time parameter within the tile provider.
+        var parameters = {};
+        var keys = [];
+        if (defined(dynamicIntervalData)) {
+            if (dynamicIntervalData instanceof Object){
+                try {
+                    Object.keys(dynamicIntervalData).forEach(function (key, index) {
```

Comment:
This is just copying `dynamicIntervalData`. We can just pass that directly into `setQueryParameters`.